config/locales/en.yml

```diff
@@ -1276,9 +1276,9 @@ en:
     open_in_web: Open in web
     over_character_limit: character limit of %{max} exceeded
     pin_errors:
+      direct: Direct posts cannot be pinned
```

Comment:
Direct posts is not a term we use as such anywhere else. Direct-visibility posts might be more clear. Or mention-only posts… to be honest maybe you can keep “non-public”, it still works and sounds better than direct posts.

Reply:
This isn't actually true, we talk about “Direct messages” in the accounts auto-cleanup page, and also in the privacy policy, where we also use “Direct posts”. But I guess in this case this could be changed to something like “Posts with direct visibility cannot be pinned”

Reply:
Changed it to “Posts that are only visible to mentioned users cannot be pinned”